src/explore-education-statistics-common/src/modules/release/components/__tests__/NationalStatisticsSection.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import NationalStatisticsSection from '@common/modules/release/components/NationalStatisticsSection';
+
+describe('NationalStatisticsSection', () => {
+  test('renders', () => {
+    render(<NationalStatisticsSection />);
+
+    expect(
+      screen.getByRole('link', {
+        name: 'Office for Statistics Regulation',
+      }),
+    ).toBeInTheDocument();
+  });
+
+  test('shows the heading if showHeading is true', () => {
+    render(<NationalStatisticsSection showHeading />);
+
+    expect(
+      screen.getByRole('heading', { name: 'National statistics' }),
+    ).toBeInTheDocument();
+  });
+
+  test('hides the heading if showHeading is false', () => {
+    render(<NationalStatisticsSection showHeading={false} />);
+
+    expect(
+      screen.queryByRole('heading', { name: 'Official statistics' }),
+    ).not.toBeInTheDocument();
+  });
+
+  test('includes the OSR guidance text', () => {
+    // Introduced because of changes to the National Statistics Designation Review,
+    // Documented in https://dfedigital.atlassian.net/browse/EES-4620
+
+    render(<NationalStatisticsSection />);
+
+    expect(
+      screen.getByText(
+        'Our statistical practice is regulated by the Office for Statistics Regulation (OSR).',
+      ),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.queryAllByRole('link', {
+        name: 'Code of Practice for Statistics',
+      })[0],
```

Comment:
is there a way to check all instances satisfy this check, rather than just the first one?
(I guess it also depends on the answer to my previous comment around the link looking slightly different for one of the links compared to the ticket description)

Reply:
I think it's probably fine to just check one, we don't generally test all static content is correct in tests.

Reply:
ok :)